frontend/src/routes/CriticalQuestions/index.js

```diff
@@ -177,13 +184,10 @@ function CriticalQuestions(props) {
                 </div>
                 <div className={classNames("grid-1", styles["grid-1"])}>
                     <TextField
-                        id="standard-number"
```

Comment:
Ooh, thanks for removing those old sample IDs 💕 All code cleanup is much appreciated !